react/src/components/search-app/Application.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import SearchBarComponent from './SearchBarComponent';
+import SearchResultsComponent from './SearchResultsComponent';
+
+
+class Application extends React.Component {
+
+  constructor(props) {
+    super(props);
+    this.state = { toggle: false };
+  }
+
+
+  render() {
+    const { toggle } = this.state;
+
+    return (
+      <div style={ toggle ? { backgroundColor: 'red' } : {} }>
```

Comment:
I guess you forgot to delete it after copying from the example component, but we don't need this here. After removing it, you could also make the component stateless.